src/game/game_controller.js

```diff
@@ -29,8 +29,7 @@ export function giveCommand(dirtyCommand) {
  */
 export function commandsDone() {
     if (currentCommand === undefined) {
-        console.error("current command undefined");
-        return;
+        return; //if not waiting current command, no problem, just dont postMessage
```

Comment:
update the jsdoc with this comment instead

Reply:
It was a niche case in the start of the program but good point, updated.